src/ptvsd/_vendored/pydevd/pydevconsole.py

```diff
@@ -185,36 +184,43 @@ def init_mpl_in_console(interpreter):
 
 if sys.platform != 'win32':
 
-    def pid_exists(pid):
-        # Note that this function in the face of errors will conservatively consider that
-        # the pid is still running (because we'll exit the current process when it's
-        # no longer running, so, we need to be 100% sure it actually exited).
+    if not hasattr(os, 'kill'):  # Jython may not have it.
 
-        import errno
-        if pid == 0:
-            # According to "man 2 kill" PID 0 has a special meaning:
-            # it refers to <<every process in the process group of the
-            # calling process>> so we don't want to go any further.
-            # If we get here it means this UNIX platform *does* have
-            # a process with id 0.
+        def pid_exists(pid):
```

Comment:
This is a fix in the interactive console with jython (which doesn't appear to have `os.kill`). This is unused in `ptvsd`, so, just synchronizing the code here.